cypress/e2e/smoke/footerEU.cy.js

```diff
@@ -0,0 +1,86 @@
+import '@testing-library/cypress/add-commands'
+import footerEuPage from '../../support/POM/commonEuPage';
+describe('QATEST-1422 Footer EU Responsive', () => {
+  beforeEach(() => {
+    cy.c_visitResponsive(Cypress.env('RegionEU'), 'small')
+  });
+  it('Footer logo and Social media icons verification for EU', () => {
+    footerEuPage.elements.footerLogo().should('be.visible')
+    footerEuPage.areSocialLinksVisible();
+  });
+  it('Footer Social Media Links Verification for EU', () => {
+    footerEuPage.areSocialLinksCorrect(footerEuPage.elements.footerFaceBookLogo, footerEuPage.externalEUURls.facebookEU);
+    footerEuPage.areSocialLinksCorrect(footerEuPage.elements.footerInstagramLogo, footerEuPage.externalEUURls.instagramEU);
+    footerEuPage.areSocialLinksCorrect(footerEuPage.elements.footerTwitterLogo, footerEuPage.externalEUURls.twitterEU);
+    footerEuPage.areSocialLinksCorrect(footerEuPage.elements.footerYoutubeLogo, footerEuPage.externalEUURls.youtubeEU);
+    footerEuPage.areSocialLinksCorrect(footerEuPage.elements.footerLinkedInLogo, footerEuPage.externalEUURls.linkedInEU);
+  });
+  it('Footer Menu About Us Pages verification for EU', () => {
+    footerEuPage.elements.hamburgerMenu().click();
+    footerEuPage.elements.aboutUsMenu().click();
+    footerEuPage.areAboutUsLinksVisible();
+    footerEuPage.elements.whoWeAreLink().click();
+    footerEuPage.elements.whoWeArePageText().should('be.visible');
+    footerEuPage.elements.whyChooseUsLink().click();
+    footerEuPage.elements.whyChooseUsPageText().should('be.visible');
+    footerEuPage.elements.partnershipProgrammesLink().click();
+    footerEuPage.elements.partnershipProgrammesPageText().should('be.visible');
+    cy.go(-1);
+    footerEuPage.elements.hamburgerMenu().click();
```

Comment:
footerEuPage.elements.hamburgerMenu().click() can be turned into a custom command and re used
```suggestion
    footerEuPage.elements.hamburgerMenu().click();
```

Reply:
done